edge/pkg/common/dbm/db.go

```diff
@@ -72,9 +73,22 @@ func InitDBManager() {
 	DBAccess.Using(dbName)
 }
 
+// Clean db file if it is neccessary
+func Clean() {
```

Comment:
CleanDBFile(file string) ??

Reply:
the file name is defined in db.go, what about `CleanDBFile()`?

Reply:
Done, Thanks